concepts/basics/about.md

```diff
@@ -3,7 +3,7 @@
 The Basics concept introduced [Go](https://golang.org) as a statically typed, compiled programming language. 
 The language is often referred to as Golang because of its domain name, golang.org, but the proper name is Go.
 
-The Basics concept and corresponding exercises introduced three major language features: Packages, Functions, and Variables.
+The Basics concept introduced three major language features: Packages, Functions, and Variables.
```

Comment:
I think this should be present tense, if someone reads this back after some month they might not remember that first introduction a while ago.

```suggestion
The Basics concept introduces three major language features: Packages, Functions, and Variables.
```